src/content/administration/dashboard.mdx

```diff
@@ -62,11 +62,12 @@ The `User` column is also the user's default `namespace` name/id. For example, a
 
 :::
 
-For each user, an administrator will be able to:
+On this page an administrator will be able to:
 
-- View their name, email, and avatar
-- Change their role (`User` or `Admin`)
-- Remove the user
+- View the total number of users on the Okteto instance (The `Total` value in the top right of the table)
+- View each user's name, email, and avatar
```

Comment:
Should we mention the new `Last Seen` field?

Reply:
```suggestion
- View each user's name, email, and avatar
- View when a user was `Last Seen` (any user who recently sent any API request to Okteto, e.g. from the browser, CLI, GitHub actions)
```

Ah, yes. We should! Good catch.

How does this proposed point look?